go/common/host/host.go

```diff
@@ -23,8 +23,8 @@ type Host interface {
 	SubmitAndBroadcastTx(encryptedParams common.EncryptedParamsSendRawTx) (common.EncryptedResponseSendRawTx, error)
 	// ReceiveTx processes a transaction received from a peer host.
 	ReceiveTx(tx common.EncryptedTx)
-	// ReceiveBatch processes a batch received from a peer host.
-	ReceiveBatch(batch common.EncodedBatch)
+	// ReceiveBatches receives a set of batches from a peer host.
+	ReceiveBatches(batches common.EncodedBatches)
```

Comment:
This is a set of batches now, not just one, to accommodate the case where you're requesting a historical chain (and not just the latest).